### DISCUSSION TITLE
Jetbrains Fleet

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#939] [General]
稍微用了一下，node 静态代码分析很烂，前端代码高亮基本没有，插件也没有，最离谱的是折腾了半天发现好像底下的终端栏不能最小化，只能把所有终端窗口全关闭掉才会消失

<img width="469" alt="image" src="https://user-images.githubusercontent.com/9006264/195367340-337133f3-265f-4281-a901-79f4af6ece4b.png">

<img width="1620" alt="image" src="https://user-images.githubusercontent.com/9006264/195368259-39a222cd-9fa7-463e-942e-d65322b5526a.png">

<details>
<summary>VS Code</summary>

<img width="694" alt="image" src="https://user-images.githubusercontent.com/9006264/195369304-eaaf6f96-76ad-492c-b41f-16d80c2ebce4.png">
</details>

**Comment by Arondight** [comment]
搞不好又是个 kpi 工程

**Reply by mark9804** [reply]
我觉得主要还有免费的 vscode 和它竞争（fleet 打算收费），按目前这个完成度推算想站稳除非 vscode 开倒车
